src/components/Profile/components/Identification/components/SocialMediaLinks/components/LinksDialog/index.jsx

```diff
@@ -23,9 +23,17 @@ const LinksDialog = ({ links, createSnackbar, onClose, setLinks }) => {
   };
 
   const validateField = (platform, value) => {
-    const { prefix, prefix2 } = socialMediaInfo[platform];
-    const isValid =
-      value === '' || value.indexOf(prefix) === 0 || (prefix2 && value.indexOf(prefix2) === 0);
+    const { prefix } = socialMediaInfo[platform];
+    let isValid = value === '';
+    console.log('isValid' + isValid);
+    if (!isValid && prefix) {
+      for (let i = 0; i < prefix.length; i++) {
+        if (value.indexOf(prefix[i]) === 0) {
+          isValid = true;
+        }
+      }
```

Comment:
FYI, I know this is a draft, but normal `for` loops are prone to errors, so we tend to prefer functions like `.map(`, `.filter(`, `.some(`, or `.foreach(`.

`.some(...)` checks if any of the elements in the collection return true for the logic in the lamda function.

For example:
```suggestion
      isValid = prefix.some(p =>
        value.indexOf(p) === 0);
```